changelogs/config.yaml

```diff
@@ -0,0 +1,30 @@
+---
+changelog_filename_template: CHANGELOG.rst
```

Comment:
```suggestion
changelog_filename_template: ../CHANGELOG.rst
```
This will *probably* place it one level up.

Reply:
That seemed to work.